1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,30 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  const divContent = document.getElementById("content");
+
+  const createP = document.createElement("p");
+  const createSelect = document.createElement("select");
+
+  const splitParagraph = paragraph.split(" ");
+  splitParagraph.forEach((x) => {
+    const createSpan = document.createElement("span");
+    createSpan.textContent = x + " ";
+    createSpan.onclick = (e) => {
```

Comment:
I recommend to look through [this post ](https://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick#:~:text=addEventListener%20can%20add%20multiple%20events,added%20within%20elements) to know the difference between **onClick** and **addEventListener**